pkg/serviceaccount/tokens_controller.go

```diff
@@ -44,18 +44,16 @@ type TokensControllerOptions struct {
 	// SecretResync is the time.Duration at which to fully re-list secrets.
 	// If zero, re-list will be delayed as long as possible
 	SecretResync time.Duration
-}
 
-// DefaultTokenControllerOptions returns
-func DefaultTokenControllerOptions(tokenGenerator TokenGenerator) TokensControllerOptions {
-	return TokensControllerOptions{TokenGenerator: tokenGenerator}
+	ClientCA string
```

Comment:
actually, take an interface that can give you a `[]byte`, so you don't have to restart the controller to rotate a CA